server/master/src/main/java/org/apache/accumulo/master/tableOps/tableImport/CreateImportDir.java

```diff
@@ -42,26 +43,35 @@
   @Override
   public Repo<Master> call(long tid, Master master) throws Exception {
 
-    UniqueNameAllocator namer = master.getContext().getUniqueNameAllocator();
-
-    Path exportDir = new Path(tableInfo.exportDir);
     String[] tableDirs = ServerConstants.getTablesDirs(master.getContext());
 
-    log.info("Looking for matching filesystem for " + exportDir + " from options "
-        + Arrays.toString(tableDirs));
-    Path base = master.getFileSystem().matchingFileSystem(exportDir, tableDirs);
-    if (base == null) {
-      throw new IOException(tableInfo.exportDir + " is not in a volume configured for Accumulo");
-    }
-    log.info("Chose base table directory of " + base);
-    Path directory = new Path(base, tableInfo.tableId.canonical());
+    create(tableDirs, master);
+
+    return new MapImportFileNames(tableInfo);
+  }
 
-    Path newBulkDir = new Path(directory, Constants.BULK_PREFIX + namer.getNextName());
+  void create(String[] tableDirs, Master master) throws IOException {
```

Comment:
A function comment please.  In particular describe what tableDirs is coming into this function.

Reply:
Added function comment